src/components/navigation/nav-tab-panel.tsx

```diff
@@ -38,8 +38,11 @@ export class NavTabPanel extends BaseComponent<IProps, IState> {
     const { tabs } = this.props;
     const { ui, user, supports } = this.stores;
     const selectedTabIndex = tabs?.findIndex(t => t.tab === ui.activeNavTab);
+    const resourceWidth = ui.dividerPosition === 50 ? `calc(50% - 4px)`
```

Comment:
Again, with the eventual slider implementation, the magic 50 value will go away. You can write this without it to make the code more robust:
```typescript
    const resourceWidth = ui.dividerPosition === 0
                            ? 0
                            : ui.dividerPosition === 100
                                ? "calc(100% - 48px)"
                                : `calc(${ui.dividerPosition}% - 4px)`
```